Feature/hee 236 contact card

goodvibes opened the pull request:
## Description

https://manifesto.atlassian.net/browse/HEE-236

Creating FE prototype for contact component

## Checklist

- [x ] Tested against our [testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)

duncanheron commented on app/assets/components/contact/contact.js:
```diff
@@ -0,0 +1,12 @@
+export default () => {
+  // Loops through dom and finds all elements with nhsuk-contact--clickable class
```
I am not sure what this file is doing - I cant see a reference to `nhsuk-contact--clickable` anywhere else.

the comments here are a bit verbose - I think the code is clear enough without them.
Maybe a concise single comment describing the javascript if you think it needs explaining

duncanheron reviewed (commented):
Thanks looking good - one small comment from me.

smanivasag commented:
Figma https://www.figma.com/file/olZRiuJrZXfEaNeMMb7aNp/HEE-Designs?node-id=1%3A175 doesn't seem to indicate heading level for contact title. But, if we assume `<h2>` and `<h3>` heading levels for title for contact and contact card resp. then probably we may need to apply the following heading levels:

### Contact:
```
Title: <h2>
Name: <h3>
Department: <h4>
```

### Contact Card:
```
Title: <h3>
Name: <h4> or <b>
Department: <p>
```

Hope that makes sense ?

goodvibes commented:
Changed heading logic -

Heading - H2, card H3
Name - H3, card H4
Department with Name - H4, card H5
Department no Name - H3, card H4

Removed leftover js from card component

smanivasag commented:
> Changed heading logic -
> 
> Heading - H2, card H3
> Name - H3, card H4
> Department with Name - H4, card H5
> Department no Name - H3, card H4
> 
> Removed leftover js from card component

@goodvibes - Thanks for this. The look and feel of name & department both on Contact and Contact Cards looks to be similar and doesn't look to be aligned with figma. Not sure if it's to do with font-size of the classes used in those elements. Could you make sure that the elements align to figma ?

Happy to have a call if that helps.
